### DISCUSSION TITLE
Obtain workflow's output

### DISCUSSION THREAD

**Discussion by najoshi7** [PrefectHQ/prefect#3723] [Q&A]
Hello prefect community,

I know the **workflow-id** , **workflow-name** and **workflow run-id** for a particular workflow.
How to obtain output & results of a workflow in a Python program?

Thanks.

**Comment by joshmeek** [comment]
Hi @najoshi7 the results from your tasks can be accessed depending on where you choose to store them. Take a look at the results documentation 🙂  https://docs.prefect.io/core/concepts/results.html (or if you are performing in-process result access: https://docs.prefect.io/core/idioms/task-results.html)

**Reply by najoshi7** [reply]
Thanks @joshmeek
